src/EPPlus/Style/ExcelBorder.cs

```diff
@@ -128,29 +124,30 @@ internal override string Id
         /// Set the border style around the range.
         /// </summary>
         /// <param name="Style">The border style</param>
-        public void BorderAround(ExcelBorderStyle Style)
+        public void BorderAround(ExcelBorderStyle Style, bool overrideNeigbourCellBorder = false)
         {
-            BorderAround(Style, Color.Empty);
+            BorderAround(Style, Color.Empty, overrideNeigbourCellBorder);
         }
+        
         /// <summary>
         /// Set the border style around the range.
         /// </summary>
         /// <param name="Style">The border style</param>
         /// <param name="Color">The color of the border</param>
-        public void BorderAround(ExcelBorderStyle Style, System.Drawing.Color Color)
+        public void BorderAround(ExcelBorderStyle Style, System.Drawing.Color Color, bool overrideNeigbourCellBorder = false)
```

Comment:
Add xml comment for the overrideNeigbourCellBorder parameter. Maybe this parameter should be true as default?